src/modules/simulator/simulator.h

```diff
@@ -380,4 +380,5 @@ class Simulator : public ModuleParams
 		(ParamInt<px4::params::MAV_TYPE>) _param_system_type
 	)
 
+#endif
```

Comment:
This looks like it does not belong into this commit. Same for `send_controls` above.

Reply:
Yep, that appears to have been a mistake caught by a later commit.  This forces me to re-write all of the commits, should be done shortly.  You'll have to re-start your review, apologies for that.